doc/monitoring/when_use_cartridge.rst

```diff
@@ -0,0 +1,221 @@
+.. _when_use_cartridge:
+
+When use Cartridge
+==================
+
+.. _when_use_cartridge-getting_started:
+
+Getting started
+---------------
+
+Если вы используете фреймворк `Cartridge <https://www.tarantool.io/en/doc/latest/book/cartridge/>`__
+для разработки своего приложения, в него уже добавлена перманентная роль metrics, которая подключается по
```

Comment:
Well, this isn't true.

First, you still need to install the rock with `tt rocks install metrics <version>`/`tarantoolctl rocks install <version>` or by adding a `'metrics == <version>'` dependency to the rockspec and running `tt rocks make`/`tarantoolctl rocks make`/`cartridge build`. `metrics` rock is not a dependency of `cartridge` rock.

Second, as you mention it below, you need to include `cartridge.roles.metrics` role to `cartridge.cfg`. Until that, no permanent role will be enabled.

After both of the above had happened, you can say that everything should be up and running.

Reply:
For some reason I thought it was included.
I suggest then to add a separate section about the installation.